vendor/src/github.com/seccomp/libseccomp-golang/seccomp.go

```diff
@@ -20,7 +20,7 @@ import (
 
 // C wrapping code
 
-// #cgo LDFLAGS: -lseccomp
+// #cgo pkg-config: libseccomp
```

Comment:
Hmm, perhaps "#cgo pkg-config" is not working at all with gccgo:  golang/go#11739